### PR TITLE
refactor(gatsby-transformer-remark): Refactors out Bluebird usage in transformer-remark

### PR DESCRIPTION
**Pull request by lourd**
Just some minor cleanup, noticed the usage of Bluebird's `Promise.each` was unnecessary when reading through the source.

### CODE REVIEW

File: packages/gatsby-transformer-remark/src/extend-node-type.js [hunk 1]
```diff
@@ -193,8 +192,8 @@ module.exports = function remarkExtendNodeType(
       if (process.env.NODE_ENV !== `production` || !fileNodes) {
         fileNodes = getNodesByType(`File`)
       }
-      // Use Bluebird's Promise function "each" to run remark plugins serially.
-      await Promise.each(pluginOptions.plugins, plugin => {
+      // Use a for loop to run remark plugins serially.
+      for (const plugin of pluginOptions) {
```
Comment (ascorbic) [inline packages/gatsby-transformer-remark/src/extend-node-type.js:196]:
```suggestion
      for (const plugin of pluginOptions.plugins) {
```

**Comment by ascorbic** [comment]
Hi. Thanks for this. This is failing tests right now. I _think_ this is because of an error in the code. I've added a suggestion. Could you take a look and see if it fixes it? You can run `yarn jest` in the root of the monorepo to check.

File: packages/gatsby-transformer-remark/src/extend-node-type.js (hunk 1 above)
Comment (lourd) [reply packages/gatsby-transformer-remark/src/extend-node-type.js]: 🤦 👍

**Comment by lourd** [comment]
Updated with your suggestion and merged with master. I don't think the still-failing tests are related to this change?

**Review by wardpeet — APPROVED** [review]
Thanks!

**Pull request merged**